src/ignitus-Shared/Components/ProfileCard/Styles/index.scss

```diff
@@ -0,0 +1,223 @@
+$base-color: #000066;
+$white-bg: #fff;
+$link-color: #939393;
+
+.list-item {
+  padding: 2em 1em;
+  display: flex;
+  flex-wrap: wrap;
+  box-shadow: 0 1px 8px 1px rgba(0, 0, 0, 0.1);
+  border-radius: 10px;
+  position: relative;
+  background-color: $white-bg;
+  color: $base-color;
+}
+
+.profile-details {
+  padding-left: 150px;
+  padding-top: 20px;
+  flex-basis: 50%;
+  max-width: 50%;
+  position: relative;
+}
+
+.avatar-div {
+  left: 0;
+  position: absolute;
+  top: 0;
+  height: 100%;
+  margin-right: 20px;
+  width: 150px;
+}
+
+.profile-avatar {
+  left: 50%;
+  position: absolute;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  width: 120px;
```

Comment:
Same here Please don't use `position: absolute;  left: 0; top: 10px;` I don't understand their use-case they are generally helpful in layering content over each other like if we have to use `z-index` etc.